backend/schema.js

```diff
@@ -189,6 +189,67 @@ input PhotoInput {
     mediaType: String!
 }
 
+type Portfolio {
+    id: ID!
+    portfolioPeriod: PortfolioPeriod
+    student: User
+    yearLevel: String
+    academicProgram: String
+    pieces: [Piece]
+    createdAt: Date!
+    updatedAt: Date!
+}
+
+interface Piece {
```

Comment:
interface so that it can get implemented by other types. Follows Entry's pattern